presto-main/etc/catalog/pinot.properties

```diff
@@ -0,0 +1,7 @@
+connector.name=pinot
+
+pinot.controller-urls=localhost:9000
+
+# Infer Pinot time fields to Presto Date/Timestamp type
+pinot.infer-date-type-in-schema=true
+pinot.infer-timestamp-type-in-schema=true
```

Comment:
So you think that these should continue to remain as default false ? What Pinot version were these introduced in ? Is it >= 0.4.0 ? (ie were they introduced in the same version as the SQL end point).

Reply:
They are introduced in Pinot 0.3.0, just if we turn these configs off, those fields will be treated as INT or LONG or VARCHAR. 

I’m also fine to make them default to be true.